tests/xfstests/run.pm

```diff
@@ -143,7 +143,8 @@ sub run {
         $test =~ s/\//-/;
         $targs->{name} = $test;
         $targs->{enable_heartbeat} = $enable_heartbeat;
-        $targs->{lastone} = 0;
+        $targs->{last_one} = 0;
+        $targs->{my_instance} = $args->{my_instance} if is_public_cloud;
```

Comment:
Publiccloud tests need also `provider`: where is it set?

Reply:
For the subtest, in case it fails, only the instance is required because it should be rebooted then.